src/OrchardCore.Modules/Orchard.OpenId/README.md

```diff
@@ -103,3 +103,25 @@ OpenID Connect apps require the following configuration.
       "AllowHybridFlow": false
 }
 ```
+
+## Configuring Certificates
+
+### Windows / IIS
+
+Several tools are availble for generating a signing certificate on Windows and/or IIS, for example:
+
++ IIS Server Manager _(offers not much control)_
+    1. Server Certificates
+    2. Create Self-Signed Certificate 
++ PowerShell _(offers full control)_
+    1. `New-SelfSignedCertificate` https://technet.microsoft.com/en-us/itpro/powershell/windows/pkiclient/new-selfsignedcertificate
```

Comment:
Sounds like a good idea (even if the signing certificate is not used for SSL/TLS).

It reminds me I need to add a check in the OIDC server middleware to throw an exception when trying to register an outdated certificate.